app/internal_packages/notifications/specs/update-notification-spec.tsx

```diff
@@ -81,6 +81,14 @@ describe('UpdateNotification', function describeBlock() {
         notif.find('#action-0').simulate('click'); // Expects the first action to be the install action
         expect(ipcSendArgs).toEqual(['command', 'application:install-update']);
       });
+
+      it('should dismiss the update notification prompt', () => {
```

Comment:
Thanks for contributing a test! I'm going to rebase this on master and I'll see why those other test failures are happening.